apps/saleor-app-checkout/README.md

```diff
@@ -37,27 +37,36 @@ To develop `saleor-app-checkout` app locally, you need to create a tunnel for it
 To create a tunnel, run:
 
 ```bash
-npx saleor app tunnel 3001
+npx saleor app tunnel 3001 --force-install
```

Comment:
```suggestion
npx saleor app tunnel 3001
```

Force install is used if your app is already installed and you would like to install it again.
Installation is automatic on the first try